benchmarks/performance-poetry/complex-poetry/src/androidTest/java/com/squareup/benchmarks/performance/complex/poetry/RenderPassTest.kt

```diff
@@ -174,7 +174,7 @@ class RenderPassTest {
       scenario: String,
       by: String? = null
     ) = fail(
-      "Uh Oh! You have worsened the $subject ${by?.let { "by $by " }}in $scenario!" +
+      "Uh Oh! You have worsened the $subject ${by?.let { "by $by " } ?: ""}in $scenario!" +
         " The value is now $value. This likely results in worse performance." +
```

Comment:
For each of these messages, I found myself wishing it showed both the before and the after.

```suggestion
        " The value is now $value (was $oldValue}). This likely results in worse performance." +
```

Reply:
Thanks ya - that makes sense.